client/state/timezones/reducer.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import {
+	DESERIALIZE,
+	SERIALIZE,
+
+	TIMEZONES_RECEIVE,
+	TIMEZONES_REQUEST,
+	TIMEZONES_REQUEST_FAILURE,
+	TIMEZONES_REQUEST_SUCCESS,
+} from 'state/action-types';
+import {
+	rawOffsetsSchema,
+	timezonesSchema
+} from './schema';
+
+const offsets = createReducer( [], {
+	[ TIMEZONES_RECEIVE ]: ( state, { rawOffsets = [] } ) => rawOffsets.slice( 0 )
+}, rawOffsetsSchema );
+
+const timezones = createReducer( {}, {
+	[ TIMEZONES_RECEIVE ]: ( state, { timezonesByContinent } ) => ( timezonesByContinent )
+}, timezonesSchema );
+
+export const items = combineReducers( {
```

Comment:
> Also, should we track the individual timezone items separate from the continent groupings? What if I wanted to look up the label associated with America/Regina ? I'm not sure we have a use case for this, but would be a little more cumbersome with this structure if we wanted to.

Yes, but it's the other side of the same coin. With this arrangement is pretty simple building the timezone selectors grouped by continent.

<img src="https://cloud.githubusercontent.com/assets/77539/22439154/62bc99e2-e70d-11e6-8793-51487e2e3024.png" width="200px" />

We could change this whenever we want. It's up to us. In fact, the `/timezones` endpoint gives us the flatten timezones object in its response.

<img src="https://cloud.githubusercontent.com/assets/77539/22439268/b7a5bf24-e70d-11e6-9f6c-aae9f564e6b4.png" width="300px" />